### PR TITLE
Q4 comment count added

### PR DESCRIPTION
**Pull request by Luke95G**

### CODE REVIEW

File: __tests__/app.test.js [hunk 1]
```diff
@@ -32,7 +33,39 @@ describe("app", () => {
...
+          expect(review).toHaveProperty("owner");
+          expect(review).toHaveProperty("review_img_url");
+          expect(review).toHaveProperty("review_body");
+          expect(review).toHaveProperty("category");
+          expect(review).toHaveProperty("created_at");
+          expect(review).toHaveProperty("votes");
+          expect(review).toHaveProperty("comment_count");
+        });
+      });
+  });
```
Comment (elijaharley) [inline __tests__/app.test.js:59]: I'd consider adding the expect(objectContaining) syntax here to ensure that both keys and values are as expected eg 
reviews.forEach((review) => {
          expect(review).toEqual(
            expect.objectContaining({
             "title": expect.any(String)...etc

File: __tests__/app.test.js [hunk 2]
```diff
@@ -32,7 +33,39 @@ describe("app", () => {
...
+      .get("/api/reviews")
+      .expect(200)
+      .then((result) => {
+        const input = result.body.reviews;
+        expect(input).toBeSorted("created_at", {
+          descending: true,
+        });
+      });
   });
 });
```
Comment (elijaharley) [inline __tests__/app.test.js:71]: Nice implementation of jest sorted here.

File: controller.js [hunk 3]
```diff
@@ -1,16 +1,20 @@
...
-const getAllReviews = (request, response) => {};
+const getAllReviews = (request, response, next) => {
+  readReviews()
+    .then((reviews) => {
+      response.status(200).send({ reviews });
+    })
+    .catch(next);
+};
 
 module.exports = { getAllCategories, getAllReviews };
```
Comment (elijaharley) [inline controller.js:20]: Good addition of catch blocks

File: model.js [hunk 4]
```diff
@@ -8,7 +8,16 @@ readCategories = () => {
...
+  FROM reviews
+  LEFT JOIN comments ON reviews.review_id = comments.review_id
+  GROUP BY reviews.review_id
+  ORDER BY created_at DESC; `;
+  return db.query(queryString).then((results) => {
+    return results.rows;
+  });
 };
 
 module.exports = { readCategories, readReviews };
```
Comment (elijaharley) [inline model.js:23]: Well done with this tricky queryString

**Review by elijaharley — APPROVED** [review]
Nice work on this one, Luke - see attached comments, good to merge.

File: __tests__/app.test.js (hunk 1 above)
Comment (Luke95G) [reply __tests__/app.test.js:59]: Just added this ^^! Was getting false positives for ages lol oops but all good now! Thanks for the feedback :)

**Pull request merged**